ui/packages/components/src/EventDetails/EventDetails.tsx

```diff
@@ -70,6 +71,10 @@ export function EventDetails({
     batch = events;
   }
 
+  const isInternalEvent = useMemo(() => {
+    return Boolean(singleEvent?.name?.startsWith('inngest/'));
+  }, [singleEvent]);
+
```

Comment:
Nit: `useMemo` is overkill here

```suggestion
  const isInternalEvent = Boolean(singleEvent?.name?.startsWith('inngest/'));
```